envision/client.py

```diff
@@ -84,7 +84,7 @@ def __init__(
         if sim_name:
             # String length limit for display is 20 characters
             # Replace all special (non-alphanumeric) characters to "_" to avoid invalid key values
-            sim_name = re.sub("\W+", "_", sim_name[:20])
+            sim_name = re.sub(r"\W+", "_", sim_name[:20])
```

Comment:
"Note that any invalid escape sequences in Python’s usage of the backslash in string literals now generate a `DeprecationWarning` and in the future this will become a `SyntaxError`. The solution is to use Python’s raw string notation for regular expression patterns."

https://docs.python.org/3/library/re.html